routes/[version]/apikey.js

```diff
@@ -0,0 +1,45 @@
+const crypto = require('crypto');
+const { checkAuth } = require('../../middlewares/auth');
+const { selectAPIToken, setAPIToken, removeAPIToken } = require('../../services/db');
+
+module.exports = {
+    get: [checkAuth, get],
+    put: [checkAuth, put],
+    delete: [checkAuth, del],
+};
+
+const generateAPIToken = (length = 11) => 'METIS_' + crypto.randomBytes(length).toString("hex"); // 28 chars
```

Comment:
@knopki @pyoner I use this in favor of UX: the key should not be too long plus it should contain alphanumeric symbols only in order to be copied with the mouse easily (double click copies the entire string, not the part of it)